hubitat.py

```diff
@@ -24,8 +25,17 @@ def __init__(self, conf: dict):
         if not self.device_groups:
             raise Exception("At least one device group must be specified in the config file.")
 
-    def resolve_device(self, name: str, device_groups: list[DeviceGroup]) -> Device:
-        return self._aliases.resolve("device", name, lambda name: self.get_device(name, device_groups))
+    def resolve_devices(self, names: str, device_groups: list[DeviceGroup]) -> list[Device]:
+        devices = set()
+        for name in names.split(self._device_name_separator):
+            name = name.strip()
+            if not name:
+                continue
+            device = self._aliases.resolve("device", name, lambda name: self.get_device(name, device_groups))
```

Comment:
And resolve() already takes care of multiple spaces between parts of the name, right?

Reply:
Hubitat allows for multiple spaces between portions of a device name or network Id, so the bot removing these would be a bug. 
Side note: Hubitat *also* allows for trailing and leading spaces, but this was already broken prior to these changes since Telegram removes leading spaces after a command.